libraries/AP_Soaring/AP_Soaring.h

```diff
@@ -86,7 +85,7 @@ class SoaringController {
     };
 
     enum class ActiveStatus {
-        DISABLED,
```

Comment:
Curious, is this rename due to the namespace problems with Rover?

Reply:
I'm not sure. Without this name change, I get:

`../../ArduPlane/RC_Channel.cpp: In member function ‘void RC_Channel_Plane::do_aux_function_soaring_3pos(RC_Channel::AuxSwitchPos)’:
../../ArduPlane/RC_Channel.cpp:84:86: error: expected unqualified-id before numeric constant
     SoaringController::ActiveStatus desired_state = SoaringController::ActiveStatus::DISABLED;
                                                                                      ^
compilation terminated due to -Wfatal-errors.`

Looks like DISABLED has been expanded as a macro to 0?